server/app/boot_jobs.rb

```diff
@@ -4,6 +4,8 @@
 
 unless ENV['RACK_ENV'] == 'test' || ENV['NO_MONGO_PUBSUB']
   MongoPubsub.start!(PubsubChannel)
+  RpcServer.supervise(as: :rpc_server)
```

Comment:
Could this be bundled to `JobSupervisor` ?

Reply:
Yes... although maybe `WorkerSupervisor` is vaguely more appropriate?

I'm also fixing the `MongoPubsub.start!`... the way it uses the Celluloid `@supervisor` is rather broken ATM.

Reply:
`WorkerSupervisor` has pools, but ofc this could be different worker.